### PR TITLE
Branch v1.4 implementation standardise

### PR DESCRIPTION
**Pull request by Droffilc13**
Add acknowledgements
Correct inconsistent format

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -431,7 +432,7 @@ persons with the specified module field, it will return an error to the user.
 
 </div>
 
-The following sequence diagram shows how the removeall command is executed:
+The following sequence diagram shows how the `removeall` command is executed:
```
Comment (qinguorui2001) [inline docs/DeveloperGuide.md:435]: no removeall command

**Review by qinguorui2001 — CHANGES_REQUESTED** [review]
little change

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Droffilc13) [reply docs/DeveloperGuide.md:435]: good point

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -432,7 +432,7 @@ persons with the specified module field, it will return an error to the user.
 
 </div>
 
-The following sequence diagram shows how the `removeall` command is executed:
+The following sequence diagram shows how the `clean` command is executed:
 <p align="center">
   <img src="images/GiveSequenceDiagram.png">
```
Comment (qinguorui2001) [inline docs/DeveloperGuide.md:437]: but actually, the below is give command? need to replace the diagram

**Review by qinguorui2001 — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Droffilc13) [reply docs/DeveloperGuide.md:437]: done

**Review by qinguorui2001 — APPROVED** [review]
lgtm

**Pull request merged**